packages/big-design/src/components/Table/TablePagination/TablePagination.tsx

```diff
@@ -6,24 +6,10 @@ import { TablePaginationProps } from '../types';
 import { StyledPaginationContainer } from './styled';
 
 export const TablePagination: React.FC<TablePaginationProps> = memo(
-  ({
-    currentPage,
-    itemsPerPage,
-    itemsPerPageOptions,
-    onItemsPerPageChange,
-    onPageChange,
-    totalItems,
-  }) => {
+  (props: TablePaginationProps) => {
```

Comment:
Can you revert this back? We actually did that to exclude the margin props to enforce design consistency. (Maybe add a comment about that?